docs/output.md

```diff
@@ -6,15 +6,375 @@ This document describes the output produced by the pipeline. Most of the plots a
 
 The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
-
 ## Pipeline overview
 
 The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes data using the following steps:
 
+Initial clustering:
+
+- [MMseqs2](#mmseqs2) initial clustering of input amino acid sequences and filtering with membership threshold
+
+Multiple sequence alignment:
+
+- [FAMSA](#famsa) aligner option. Best speed and sensitivity option to build seed multiple sequence alignment for the families
+- [mafft](#mafft) aligner option. Fast but not as sensitive as FAMSA to build seed multiple sequence alignment for the families
+- [ClipKIT](#clipkit) to optionally clip gapped portions of the multiple sequence alignment (MSA)
```

Comment:
These guidance (best speed, fast but not sensitive) sentences should really go in usage-  you want to to know this before you run the pipeline.